.buildkite/pipeline.yml

```
@@ -2,7 +2,13 @@ steps:
   - label: ":shell: Shellcheck"
     plugins:
       - shellcheck#v1.3.0:
-          files: hooks/**
+          files:
+            - hooks/**
+            - commands/**
+            - libs/**
```

There is a typo here, it should be:
```suggestion
            - lib/**
```